prac_05/wimbledon.py

```diff
@@ -0,0 +1,56 @@
+"""CP1401, Prac05, Wimbledon program to process the data and display processed information."""
+
+
+def main():
+    """Main function to process data from file."""
+    file_path = 'wimbledon.csv'  # Modify with your file's path
+    champions_data = load_data(file_path)
+
+    champion_counts = count_champions(champions_data)
+    countries = get_countries(champions_data)
+
+    # Display results
+    display_results(champion_counts, countries)
+
+
+def load_data(file_path):
+    """ Reads the file line by line and splits the data by commas."""
+    # Assuming the file contains data in the format: "Year, Champion, Country"
+    champions_data = []
+    with open(file_path, 'r') as file:
+        for line in file:
+            line = \
```

Comment:
Don't think you meant to include 'line = \' or the indentation here.

Reply:
Ahh yes i wasn't sure if i was meant to have that but thank you for the clarification.